python/tvm/auto_scheduler/measure.py

```diff
@@ -87,6 +120,24 @@ def __init__(self, task, state):
         state = state if isinstance(state, StateObject) else state.state_object
         self.__init_handle_by_constructor__(_ffi_api.MeasureInput, task, state)
 
+    def serialize(self):
```

Comment:
Can we use `__getstate__`?

Reply:
There was some weird initialization bug with `__getstate__` that I could not figure out. I'll add a comment about this.